localstack/services/awslambda/lambda_executors.py

```diff
@@ -200,6 +206,12 @@ def _execute(self, func_arn, func_details, event, context=None, version=None):
             environment['AWS_LAMBDA_FUNCTION_VERSION'] = context.function_version
             environment['AWS_LAMBDA_FUNCTION_INVOKED_ARN'] = context.invoked_function_arn
 
+        java_opts = ''
+        if config.JAVA_OPTS:
+            address = ',address=%s' % self.port
+            java_opts = config.JAVA_OPTS + address
```

Comment:
What exactly are we expecting to be the content of `config.JAVA_OPTS` here? This code doesn't seem to work with arbitrary JVM options like, say, `JAVA_OPTS=-Xmx512M`.

I think we need a bit more logic here to properly prepare the `java_opts` variable.

(ideally, we could put it into a small util function in this class, and add some unit tests for it. but not strictly necessary.)

Reply:
@whummer Great, yes, make sense. I did focus in debug option. 

Is this java_opts only used in this class? That is, can another class use it?

Reply:
> Is this java_opts only used in this class?

As far as I can see, it would only be used in this class, yes. Would be good to add it to a separate function, though, to make it easily reusable, and easily unit testable.